evalml/pipelines/utils.py

```diff
@@ -241,7 +241,7 @@ def _get_decomposer(X, y, problem_type, estimator_class, sampler_name=None):
                 y,
                 rel_max_order=order,
             )
-            if seasonal_period is not None:
+            if seasonal_period is not None and seasonal_period <= 1000:
```

Comment:
in the `make_pipeline` tests can you make sure that the decomposer is not added if the seasonal period is > 1000? I think we can just add a case here by mocking `seasonal_period`.

Reply:
Looking back, I realized I completely forgot about the `test_make_pipeline_controls_decomposer_time_series` - I added a case in there!

Reply:
Ugh, my only issue with this is the hardcoded 1000 period.  Particularly the ramifications for if a user does show up with a super high frequency dataset like the one that triggered this.

Reply:
Very fair. I'll drop it over to be a constant.